Makefile

```diff
@@ -52,20 +55,20 @@ mostlyclean:
 	rm -rf $(ROMS) $(OBJS) $(ROMS:.gb=.sym) $(ROMS:.gb=.map) $(CORRECTEDROMS)
 	find . \( -iname '*.d' \) -exec rm {} +
 
-$(ROMS): $(OBJS)
+$(ROM): $(OBJS)
 	$(RGBLINK) -d -n $(@:.gb=.sym) -m $(@:.gb=.map) -O $(BASEROM) -o $@ $^
-	$(RGBFIX) -f lh -k 01 -l 0x33 -m 0x03 -p 0 -r 3 -t "POKEMON2GOLD" $@
+	$(RGBFIX) -f lh -k 01 -l 0x33 -m 0x03 -p 0 -r 3 -t "POKEMON2$(GAME_VERSION)" $@
+	-$(PYTHON3) sort_sym.py $(@:.gb=.sym)
 
-$(CORRECTEDROMS): %-correctheader.gb: %.gb
+$(BUILD)/shim.asm: $(SHIM) | $$(dir $$@)
+	tools/make_shim -w $< > $@
+$(CORRECTEDROM): %-correctheader.gb: %.gb
 	cp $< $@
-	$(RGBFIX) -f h -m 0x10 $@
-
-$(BUILD)/shim.asm: tools/make_shim.py shim.sym | $$(dir $$@)
-	$(PYTHON) tools/make_shim.py -w -- $(filter-out $<, $^) > $@
+	$(RGBFIX) -f hg -m 0x10 $@
 
 .PHONY: coverage
-coverage: $(ROMS)
-	$(PYTHON) tools/disasm_coverage.py -m $(ROMS:.gb=.map) -b 0x40
+coverage: $(ROM)
+	$(PYTHON3) tools/disasm_coverage.py -m $(ROM:.gb=.map) -b 0x40
```

Comment:
You've missed some discussion be we are trying to stick with version agnostic python

Reply:
Clearly not doing a good job, the `encoding` keyword to `open` is a Python 3 exclusive.